src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -415,6 +411,92 @@ MatrixHeader.contextTypes = {
     navigate: PropTypes.func,
 };
 
+const ClickableThumbnail = (props) => {
+    const expandedThumbnail = props.expandedThumbnail;
+    const expandThumbnail = props.expandThumbnail;
```

Comment:
should this be called something like "expand"? The function names gets reading in the mind-set of working with thumbnails.

Reply:
I've renamed "expandThumbnail" to be "toggleThumbnail", hopefully this is clearer.